src/ComparedQueryable.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
I made this project a classic .NET project because CoreFx couldn't handle the files from System.Linq.Queryable.